lms/djangoapps/course_home_api/outline/tests/test_view.py

```diff
@@ -261,6 +261,7 @@ def test_proctored_exam(self, mock_summary):
             is_onboarding_exam=False,
         )
         sequence.is_proctored_exam = True
+        update_outline_from_modulestore(course.id)
```

Comment:
I believe because they aren't doing a publish step - which is when outlines normally get generated.

It's possible if LS becomes integral to more than just a few views that feed the MFE, we may want to change the course factory to generate them as a side effect or something. But for now, it's not that many tests that actually care.